apiary.apib

```diff
@@ -321,48 +309,64 @@ curl http://140.116.247.120:5000/participant -X GET -H "Content-Type: applicatio
 </pre>
 
 + Request (application/json)
-
     + Headers
 
             API-version: 0.1
             Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR - Authentication Bearer Token
 
 + Response 200 (application/json)
-
     + Attributes (array, fixed-type)
         + (object)
             + id: `7e54877b-3743-4d20-bbfc-5b4deb131116` - User's UUID
-            + name: BEMS1
+            + name: `BEMS1`
 
-## Electricity History Usage [/power_info]
+## Power Info [/power_info]
 
-Retuen Electricity History Usage.
+Return Electricity History Usage.
 
-### Electricity History Usage [GET /power_info{?time}]
+### PowerData (Data Table Mode) [GET /power_info{?per_page}{?page}{?time}]
 <pre>
-curl http://140.116.247.120:5000/power_info -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
-curl http://140.116.247.120:5000/power_info?time=2019-10-19 -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
+curl http://140.116.247.120:5000/power_info?per_page=10&page=1&time=2020/02/14 -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
 </pre>
 
 + Parameters
-    + time: `2019-10-19` (optional) - Query date. Format: `YYYY-mm-DD`
-        + Default: System date
+    + per_page: `10` (number, optional) - Power data per page.
+    + page: `1` (number, optional) - Power data at page.
+    + time: `2020/02/14` (string, optional) - Date for the data, default is now, format YYYY/MM/DD.
 
 + Request (application/json)
-
     + Headers
 
             API-version: 0.1
             Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR - Authentication Bearer Token
 
 + Response 200 (application/json)
+    + Attributes
+        + data (array) - 頁面所顯示的電力資訊
+            + (PowerEntry1) - 第一筆資料
+            + (PowerEntry2) - 第二筆資料
+        + page: `1` (number) - 所在頁數
+        + totalCount: `785` (number) - Query 時間內資料總數
 
+### PowerData (Data Chart Mode) [GET /power_info{?start_time}{?end_time}]
+<pre>
+curl http://140.116.247.120:5000/power_info?start_time=2020/02/14?end_time=2020/02/15 -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
+</pre>
+
++ Parameters
+    + start_time: `2020/02/14` (string, optional) - Date for the data, format YYYY/MM/DD.
+    + end_time: `2020/02/15` (string, optional) - Date for the data, format YYYY/MM/DD.
```

Comment:
These two parameters must be required to `PowerData Chart Mode`

Reply:
Noted.